packages/squidex/schema/schemas/research-outputs.json

```diff
@@ -430,6 +430,27 @@
           "isHalfWidth": false
         }
       },
+      {
+        "name": "accession",
+        "isHidden": false,
+        "isLocked": false,
+        "isDisabled": false,
+        "partitioning": "invariant",
+        "properties": {
+          "fieldType": "String",
+          "pattern": "^(\\w+\\d+(\\.\\d+)?)|(NP_\\d+)$",
```

Comment:
In the ticket it says that should match one of two regexes (think this is just the first one), could combine them into one big one with an |

Reply:
i am aware, but like i mentioned in the ticket - it looks like every provided example passes the first one. It appears that the second one is a subset of the first pattern and so I think that unless we can come up with an example of an ID that only passes the second regex we shouldnt include it

Reply:
OK cool